ddtrace/tracer/span.go

```diff
@@ -319,9 +319,57 @@ func (s *span) finish(finishTime int64) {
 		// not sampled by local sampler
 		return
 	}
+	if t, ok := internal.GetGlobalTracer().(*tracer); ok {
+		// we have an active tracer
+		feats := t.features.Load()
+		if feats.Stats && shouldComputeStats(s) {
+			// the agent supports computed stats
+			var statusCode uint32
+			if sc, ok := s.Meta["http.status_code"]; ok && sc != "" {
+				if c, err := strconv.Atoi(sc); err == nil {
+					statusCode = uint32(c)
+				}
+			}
+			t.stats.In <- &spanSummary{
+				Start:      s.Start,
+				Duration:   s.Start,
+				Name:       s.Name,
+				Resource:   s.Resource,
+				Service:    s.Service,
+				Type:       s.Type,
+				Hostname:   s.Meta["keyHostname"],
+				Synthetics: strings.HasPrefix(s.Meta[keyOrigin], "synthetics"),
+				Env:        s.Meta[ext.Environment],
```

Comment:
same think for the env, I believe it should come from env variables?

Reply:
Same answer as hostname...